README.md

```diff
@@ -1 +1,66 @@
-# First-capstone-project
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Marvel Divide 2015
+
+![Marvel Divide 2015 logo](./Images/logo/marvel-logo_250x150.gif)
+
+> This is an educational project to make a website for a conference event with home page and about page.
+
+## Built With
+
+- HTML5, CSS3, JavaScript
+- Linters:
+  - Stylelint
+  - Lighthouse
+  - Webhint
+  - ESLint
+
+## Project Documentation
+
+- Here is the [link]() to my loom video
+
+## Website Mockup 📱 💻 🖥️
+
+![mobile-look](./Images/mobile.gif)
+![Laptop-look](./Images/laptop.gif)
+
+## Live Demo
+
+[Capstone I : Marvel Divide 2015]()
```

Comment:
- [ ] [OPTIONAL] It would be better if you could deploy your project 😃

Reply:
would do that